registry/storage/driver/gcs/gcs.go

```diff
@@ -274,7 +288,7 @@ func (d *driver) GetContent(context context.Context, path string) ([]byte, error
 // This should primarily be used for small objects.
 func (d *driver) PutContent(context context.Context, path string, contents []byte) error {
 	return retry(func() error {
```

Comment:
Side comment: I dont think it's a good idea to do retries here. For consistency's sake (with S3 driver), I would remove this from here.